test/regression/simple-linear-regression.js

```diff
@@ -49,5 +51,17 @@ describe('Simple linear regression', function () {
         regression.computeX(5).should.equal(2.5);
         regression.computeX(9).should.equal(0.5);
         regression.computeX(-12).should.equal(11);
+
+        regression.toString(3).should.equal('y = -2.00x + 10.0');
+    });
+    it('SLR constant', function () {
```

Comment:
we can refactor the tests so that each assertion is in a different `it` but that's for another PR if we want.